terraform/alb/variables.tf

```diff
@@ -0,0 +1,2 @@
+variable "projectname" {type = string}
+variable "ami" {type=string}
```

Comment:
you can get the ami via data resource

```
data "aws_ami" "amzn" {
  most_recent = true
  owners      = ["amazon"]

  filter {
    name   = "name"
    values = ["amzn-ami-*"]
  }

  name_regex = ".+-amazon-ecs-optimized$"
}
```

then `image_id      = "data.aws_ami.amzn.image_id`

Reply:
thanks, i'll be using that next time
i removed ami from this module, as i'm not using this variable anymore